kratos/input_output/vtu_output.cpp

```diff
@@ -0,0 +1,1276 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+// System includes
+#include <string>
+#include <vector>
+#include <iomanip>
+#include <numeric>
+
+// External includes
+
+// Project includes
+#include "containers/container_expression/expressions/expression.h"
+#include "containers/container_expression/expressions/expression_iterator.h"
+#include "containers/container_expression/expressions/literal/literal_flat_expression.h"
+#include "containers/container_expression/specialized_container_expression.h"
+#include "includes/data_communicator.h"
+#include "includes/define.h"
+#include "includes/io.h"
+#include "utilities/global_pointer_utilities.h"
+#include "utilities/parallel_utilities.h"
+#include "utilities/pointer_communicator.h"
+#include "utilities/reduction_utilities.h"
+#include "utilities/string_utilities.h"
```

Comment:
I think a bunch of them are not used or already in the header

Reply:
removed and moved :D